cpp/oneapi/dal/io/csv/BUILD

```diff
@@ -0,0 +1,16 @@
+package(default_visibility = ["//visibility:public"])
+load("@onedal//dev/bazel:dal.bzl", "dal_module", "dal_public_includes")
+
+dal_public_includes(
+    name = "public_includes",
+)
+
+dal_module(
+    name = "csv",
+    dpc = True,
+    auto = True,
+    deps = [
+        "@onedal//cpp/oneapi/dal:common",
```

Comment:
`core` already contains `common`, so can be removed

Reply:
Fixed.